sdcm/cluster.py

```diff
@@ -3421,6 +3422,12 @@ def terminate_node(self, node: BaseNode, scylla_shards: int = 0) -> None:
             ))
         if node in self.nodes:
             self.nodes.remove(node)
+        ScyllaLogCollector.cluster_log_type = node.name
+        log_links = ScyllaLogCollector([node],
+                                       test_id=self.test_config.test_id(),
+                                       storage_dir=os.path.join(self.logdir, "collected_logs"),
+                                       params=self.params).collect_logs()
+        self.test_config.argus_client().submit_sct_logs([LogLink(log_name=node.name, log_link=log_links[0])])
```

Comment:
why do we need to report to argus at this point ? isn't enough that we have the files in place, for the collection phase to pick them up ?

i.e. i'm not sure we want to upload to s3 at this point...

Reply:
I wanted to use existing means of collecting logs which does it by default. I'm not sure yet if this is feasible yet.
Collection phase is not collecting them if collected in current form anyway, as collection dir does not match.

The one positive point about publishing it to s3 is that logs get removed right away from the sct-runner. For single node it's not taking long (db logs tar.gz usually is less than 10MB's).